crawler/crawlutils.py

```diff
@@ -458,30 +459,30 @@ def snapshot(
     signal.signal(signal.SIGHUP, signal_handler_exit)
 
     if crawlmode == Modes.OUTCONTAINER:
+	#Use custom filter logic to distribute already running containers
         containers = get_filtered_list_of_containers(options, namespace,
                                                      runtime_env)
 
+    eventQ = options['partition_strategy']['args']['eventQ']
+    delList = options['partition_strategy']['args']['delList']
+
+    if frequency < 0:
+	wait_period = 5
+    else:
+	wait_period = 5
```

Comment:
Also, a perfect spot for a ternary if:

`wait_period = 5 if frequency < 0 else 5`